### ISSUE TITLE
Removal of analysis and pathways

### ISSUE THREAD

**Issue by makingglitches** [VUE/VUE#71]
Would anyone be against removing these two obviously incomplete and not very good features aside perhaps working more on then Java class mapper ? That seems to sort of work. Anyone else care to weigh in ?

**Comment by rpavlik** [comment]
As far as I can tell, I am literally the only one who regularly responds to stuff on this repo. You can do whatever you want in your fork as long as you follow the license.

**Comment by makingglitches** [comment]
yeah underrated tool really. I used it for years though. what do other people use ? the online bs ?

**Comment by makingglitches** [comment]
Anyway what I was hoping for was a final merge in since i'm kind of sick of this roundabout  bs which is not actually helping.

This is an example of a project I'd like to see work and simplified.
I do like the java visualizer. That could be extended but there is a whole other project out there that does something similar.
